config/system/ata.yaml

```diff
@@ -16,7 +19,7 @@ switch_setup:
 
 run_trigger:
   native_vlan: 122
-  max_hosts: 8
+  max_hosts: 3
```

Comment:
For ATA, it's limiting the number of tests that can run inside of one portset -- there is always only one port-set in that case (since it's a shared network). I'd like to leave it at 3 (because that's what I've tested) until we can fix the limiting factor and then bump it up.

Reply:
Yes I understand there is only 1 portset for ata but there are 4 ports available on that portset for testing. We can leave it at 3 but 4 should be perfectly fine.